src/components/blocks/card/_card.scss

```diff
@@ -211,11 +213,11 @@
     }
   }
 
-  &:visited &__info-container {
+  &:visited {
     color: base.sitecolors-siteColor("vam-grey-3");
   }
 
-  &--picture-card:visited .b-card__info-container {
+  &--picture-card:visited {
```

Comment:
similarly this rule is better organised on line 186 (inside `&--picture-card` rule, before its `&::after` rule)